src/apify/storages/_known_actor_input_keys.py

```diff
@@ -0,0 +1,29 @@
+class _KnownInputKey(str):
+    __slots__ = ('_name',)
+
+    def __init__(self, name: str) -> None:
+        self._name = name
+
+    def __str__(self) -> str:
+        return self._name
+
+    def __repr__(self) -> str:
+        return str(self)
+
+
+class _StartUrls(_KnownInputKey):
+    url = 'url'
+    requestsFromUrl = 'requestsFromUrl'  # noqa: N815  # Intentional to respect actual naming of input keys.
```

Comment:
couldn't we use pydantic & alias field instead of "noqa: N815"?

Reply:
I got rid of the whole file. It was causing more harm than benefit